share/spice/snow/snow.css

```diff
@@ -11,6 +11,6 @@
     font-weight: 900;
 }
 
-.zci--snow .zci__header {
-    color: inherit;
+.c-base__content {
```

Comment:
whoops, you need to keep the `.zci--snow` here to namespace the CSS. Otherwise you're potentially going to change all the content on the page that matches your selector:

```css
.zci--snow .c-base__content {
   ...
}
```